core/app/helpers/spree/images_helper.rb

```diff
@@ -15,21 +15,22 @@ def spree_image_tag(image, options = {})
     end
 
     def spree_image_url(image, options = {})
+      url_helpers = options[:url_helpers] || main_app
```

Comment:
@mad-eel we don't need to pass it, we can just use `Rails.application.routes.url_helpers` directly